carmin.yaml

```diff
@@ -390,34 +429,46 @@ definitions:
       timeout:
         type: integer
         format: int64
+        description: The timeout in seconds until which the execution is killed and deleted with all its files. 0 or absense means no timeout
       status:
         type: string
-        enum: [initializing,ready,running,finished,
-        initialization_failed,execution_failed,unknown,killed]
+        enum: [initializing,ready,running,finished,initialization_failed,execution_failed,unknown,killed]
+        readOnly: true
+        description: The status of the execution. Must always be present in responses.
       inputValues:
         type: object
         additionalProperties: true
-        description: represent the input as a key/value object. The types should respect the parameters of the pipeline used for the execution
+        description:
+          Represents the input as a key/value object. The types should respect the parameters of the pipeline used for the execution.
       returnedFiles:
         type: object
         additionalProperties:
-          type: string
-          format: url
+          type: array
+          items: 
+            type: string
+            format: url
+        readOnly: true
+        description: absent when not available (e.g. execution still running). Empty if no returned file is produced
       studyIdentifier:
         type: string
         format: ascii
       errorCode:
         type: integer
         format: int64
+        readOnly: true
       startDate:
         type: integer
         format: int64
+        description: epoch in seconds
```

Comment:
I think we must indicate the reference time here. For instance : "Time in seconds since Unix epoch".